sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/KeyClient.java

```diff
@@ -1442,6 +1443,7 @@ public KeyVaultKey restoreKeyBackup(byte[] backup) {
      *
      * @return The requested number of bytes containing random values from a managed HSM.
      */
+    @ServiceMethod(returns = ReturnType.SINGLE)
     public RandomBytes getRandomBytes(int count) {
```

Comment:
Other languages moved to returning bytes directly from this method -- I remember you mentioning something particular about Java and returning byte arrays, so is that why the RandomBytes model is still being used?

Reply:
That's true, I think I missed this when going through other languages' APIViews, thanks for pointing it out!